File: android/app/src/main/java/net/ipv64/kivop/dtos/RideServiceDTOs/GetParticipantDTO.kt
package net.ipv64.kivop.dtos.RideServiceDTOs

import java.util.UUID
import java.time.LocalDateTime

data class GetParticipantDTO (
    var id : UUID?,
    var name : String,
    var driver : Boolean,
    var passengers_count : Int?,
    var latitude : Float,
    var longitude : Float,
    var itsMe : Boolean,
)

File: DTOs/Sources/RideServiceDTOs/GetParticipantDTO.swift
import Foundation

public struct GetParticipantDTO: Codable {
    public var id: UUID?
    public var name: String
    public var driver: Bool
    public var passengers_count: Int?
    public var latitude: Float
    public var longitude: Float
    public var itsMe: Bool
    
    public init(id: UUID?, name: String, driver: Bool, passengers_count: Int?, latitude: Float, longitude: Float, itsMe: Bool) {
        self.id = id
        self.name = name
        self.driver = driver
        self.passengers_count = passengers_count
        self.latitude = latitude
        self.longitude = longitude
        self.itsMe = itsMe
    }
}
